sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
@lreficent Yes you are right, the onchange does not work in the case you mentioned.

I am getting a blank recordset in `self.order_line` which is because the line is never created, so now I am not really sure how to update the requested_date field in the o2m